modules/examples/personas/deployspec.yaml

```diff
@@ -0,0 +1,26 @@
+publishGenericEnvVariables: true
+deploy:
+  phases:
+    install:
+      commands:
+      - env
+      # Install whatever additional build libraries
+      - npm install -g aws-cdk@2.126.0
```

Comment:
Not sure how much this matters but we should keep the version consistent (`2.137.0`)